tasks/main.yml

```diff
@@ -192,6 +192,17 @@
   when: cartridge_bootstrap_vshard is defined and cartridge_bootstrap_vshard
   tags: cartridge-config
 
+- name: Wait for instance to be bootstrapped
```

Comment:
Let's say "Wait for instance to be STARTED". "Bootstrapped" confuses a lot - we bootstrap vshard, cluster is bootstrapped when replicasets are created. Also we use `instance_START_timeout` here.